### PR TITLE
Amenities Pages

### PR DESCRIPTION
**Pull request by JoshuaEntrata**
# Amenities Pages

## Details

- Created initial pages for Amenities and Offers - Amenities.
- Created the Footer component.
- Added two new font styles in `tailwind.config.ts`.
- Installed `react-icons` .

## Juxtaposition

- Amenities Home Page 
image
- Function Hall Page 
image
- Swimming Pool Page 
image
- Souvenir Shop Page 
image

## Additional Notes

- `npm install`
- Follow the format of `index.tsx` files in the component directory for future PRs.
- All pages and components are not yet responsive and are currently catered for the default computer dimensions.
- The Carousel component does not yet match the one in the Figma. Will reconstruct this in the next PR.
- Need to replace the font style of body and link texts in `AmenitiesCard.tsx` with `lao-muang-dong`
- Images used are temporary.
- Used a sample video for the Amenities home page, which needs updating
- The `NewsletterInput.tsx` component is still static.

### CODE REVIEW

File: src/components/Carousel/Carousel.tsx
Comment (carlmitzchel) [inline src/components/Carousel/Carousel.tsx:1]: Implement movement indicators (Left & Right arrows) for better accessibility

**Review by carlmitzchel — CHANGES_REQUESTED** [review]

**Comment by carlmitzchel** [comment]
Lao Muang Don is integrated in PR #9 
image

File: src/components/Carousel/Carousel.tsx
Comment (JoshuaEntrata) [reply src/components/Carousel/Carousel.tsx:1]: next PR pa yung para sa carousel

**Comment by JoshuaEntrata** [comment]
### What did I do?
- Applied the updates for `lao-muang` font style in PR#9

**Review by carlmitzchel — APPROVED** [review]

**Pull request merged**